regression-tests.dnsdist/test_Protobuf.py

```diff
@@ -94,22 +173,28 @@ def checkProtobufBase(self, msg, protocol, query, initiator):
         self.assertEquals(msg.socketFamily, dnsmessage_pb2.PBDNSMessage.INET)
         self.assertTrue(msg.HasField('from'))
         fromvalue = getattr(msg, 'from')
-        self.assertEquals(socket.inet_ntop(socket.AF_INET, fromvalue), initiator)
+        self.assertEquals(socket.inet_ntop(socket.AF_INET, fromvalue), initiator)		
         self.assertTrue(msg.HasField('socketProtocol'))
         self.assertEquals(msg.socketProtocol, protocol)
         self.assertTrue(msg.HasField('messageId'))
         self.assertTrue(msg.HasField('id'))
-        self.assertEquals(msg.id, query.id)
+        self.assertEquals(msg.id, query.id)							
         self.assertTrue(msg.HasField('inBytes'))
-        self.assertEquals(msg.inBytes, len(query.to_wire()))
+        self.assertEquals(msg.inBytes, len(query.to_wire()))		
         # dnsdist doesn't set the existing EDNS Subnet for now,
         # although it might be set from Lua
         # self.assertTrue(msg.HasField('originalRequestorSubnet'))
         # self.assertEquals(len(msg.originalRequestorSubnet), 4)
         # self.assertEquals(socket.inet_ntop(socket.AF_INET, msg.originalRequestorSubnet), '127.0.0.1')
 
+
     def checkProtobufQuery(self, msg, protocol, query, qclass, qtype, qname, initiator='127.0.0.1'):
-        self.assertEquals(msg.type, dnsmessage_pb2.PBDNSMessage.DNSQueryType)
+
+	if initiator == '127.0.0.1':
+		self.assertEquals(msg.type, dnsmessage_pb2.PBDNSMessage.DNSQueryType)		# testProtobuf()
+	else:
+		self.assertEquals(msg.type, dnsmessage_pb2.PBDNSMessage.DNSResponseType)	# testLuaProtobuf()
+
```

Comment:
That blocks looks very weird because it's effectively making `checkProtobufQuery()` accept responses. Perhaps we should call `checkProtobufResponse()` when the query has been turned into a response instead?